src/components/ProductCard.tsx

```diff
@@ -1,89 +1,86 @@
-import React, { useEffect, useState } from "react";
-import { Transition } from "@headlessui/react";
+import React, { useState } from "react";
 import Image from "next/image";
 import ProudctNameAndEdition from "./ProductNameAndEdition";
 import ProductColourButton from "./ProductColourButton";
+import ProductCardShowMoreButton from "./productCardShowMoreButton";
 
 export type Content = {
   colours: string[];
   image: string[];
+  imageAlt: string;
+  index: number;
   name: string;
   edition?: string;
   price: string;
 };
 
-export default function ProductCard({ content }: { content: Content }) {
+export default function ProductCard({
+  content,
+  key,
+}: {
+  key: number;
+  content: Content;
+}) {
+
   const [selectedColour, setSelectedColour] = useState("black");
   const [cardImage, setCardImage] = useState(content.image[0]);
   const [isCardFlipped, setIsCardFlipped] = useState(true);
+  const [opacity, setOpacity] = useState(100);
```

Comment:
why is opacity set to 100 instead of just 1?

Reply:
addressed.